migtests/tests/pg/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -317,6 +318,19 @@
 					"SqlStatement": "ALTER TABLE ONLY schema2.sales_region\n    ADD CONSTRAINT sales_region_pkey PRIMARY KEY (id, region);"
 				}
 			]
+		},
+		{
+			"FeatureName": "Extensions",
```

Comment:
Can we add a test case for extensions as well?

Reply:
I was trying to add `postgis` but was facing issue related to PG version, will add one.

Reply:
Yeah, postgis can be challenging. Try and see if there is an easier alternative.

Reply:
A few others I tried are also challenging to add most of the unsupported ones are not by default available in docker PGand require installations. Should I add it later?